app/helpers/accessible_navigation_helper.rb

```diff
@@ -1,5 +1,5 @@
 module AccessibleNavigationHelper
-  def link_to_accessible(name = nil, options = nil, html_options = {}, &)
+  def govuk_link_to(name = nil, options = nil, html_options = {}, &)
```

Comment:
should we not be using the [govuk-components govuk_link_to](https://govuk-components.netlify.app/helpers/link/) helper. This looks to be implementing our own version of it